kit/src/main/java/com/oracle/javafx/scenebuilder/kit/fxom/FXOMDocument.java

```diff
@@ -443,4 +457,25 @@ public SceneGraphHolder getSceneGraphHolder() {
         public void fxomDocumentWillRefreshSceneGraph(FXOMDocument fxomDocument);
         public void fxomDocumentDidRefreshSceneGraph(FXOMDocument fxomDocument);
     }
+    
+    /**
+     * Depending on where the {@link FXOMDocument} shall be used, 
+     * it is necessary to configure the {@link FXOMDocument} creation process.
+     * The switches here can be used to configure the creation process in the desired way.
+     * This enum replaces the previously used boolean arguments in the {@link FXOMDocument} constructor.
+     */
+    public enum FXOMDocumentSwitch {
+        /**
+         * If this switch is defined, the {@link FXOMDocument} will not be normalized (see {@link FXOMNormalizer}).
+         */
+        NON_NORMALIZED,
```

Comment:
Before this PR, a boolean was used,  `normalize`. I don't see why you use the opposite enum `NON_NORMALIZED`, and then negate it `if !NON_NORMALIZED`. 

Maybe it is better to define both `NORMALIZED` and `NON_NORMALIZED`?

Reply:
I have learned, that at those places in future I'll put better an explanation why I've decided to work in a certain way, sorry for all the extra runs.